__tests__/app.test.js

```diff
@@ -73,7 +69,7 @@ describe('GET /api/articles/:article_id', () => {
         .get("/api/articles/2")
         .expect(200)
         .then(({body}) => {
-            const article = body[0];
+            const article = body.article[0];
```

Comment:
If there's only a single article, there's no need for it to be returned in an array. Please de-structure this before you send the response so you can just check body.article here.